ui/react-components/utils/AppointmentListSidePanelWrapper/getFilteredNodesOnSearch.js

```diff
@@ -0,0 +1,43 @@
+const getFilteredNodesOnSearch = (transformedTreeData, searchText) => {
+  const searchFilteredNodesList = transformedTreeData
+    .filter(element => {
```

Comment:
It will be better if we can take the extract the  function passed to `filter` as a named function. Something like `hasMatchingElementOrChildren`? Let me know your thoughts.

Reply:
Now we have refactor this function